docs/deploy.md

```diff
@@ -254,10 +193,10 @@ cf-for-k8s can be configured to [use an external database](platform_operators/ex
 1. Validate the app is reachable over **https**
 
    ```console
-   # for self-signed certs, use -k to allow insecure server connections when using SSL
-   curl -k https://test-node-app.<cf-domain>
+   curl -Lk https://test-node-app.<cf-domain>
```

Comment:
I think the `-L` usually isn't important here?

Reply:
Nope, seems like I'm wrong here. [slack thread](https://cloudfoundry.slack.com/archives/CH9LF6V1P/p1598910866049200?thread_ts=1598910828.048900&cid=CH9LF6V1P)

Reply:
The `-L` is important if you specify a URL with an `http` scheme as the ingressgateway responds with a redirect. To the best of my knowledge, it is not required for this example.